borg/testsuite/logger.py

```diff
@@ -0,0 +1,43 @@
+import logging
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
```

Comment:
no need to do that now, can be easily done IF somebody jumps on that.
personally, I am quite happy with 3.x-only.